content/tools/kops.md

```diff
@@ -16,16 +16,95 @@ assume-role
 kops create -f /conf/kops/manifest.yml
 ```
 
-## Provision Backing Service VPC
+# Upgrading `kops`
 
-## VPC peering
+To upgrade the `kops` release, there are a couple of options. First check to see if there a new release of `geodesic` with an upgraded version of `kops`. If there is one and this is the desired release, update the `FROM` image tag in the geodesic module to point to this release.
 
-# Helpful Terraform Modules
+To explictly upgrade `kops`, add this to the `Dockerfile`.
 
-- [terraform-aws-kops-route53]({{< relref "terraform-modules/kops-kubernetes/terraform-aws-kops-route53.md" >}})
-- [terraform-aws-kops-external-dns]({{< relref "terraform-modules/kops-kubernetes/terraform-aws-kops-external-dns.md" >}})
+```
+RUN make -C /packages/install kops KOPS_VERSION=1.9.0
+```
+
+Find the latest release of kops [here](https://github.com/kubernetes/kops/releases).
+
+Then follow the instructions for upgrading kubernetes.
+
+
+
+
+# Upgrading Kubernetes Release
```

Comment:
All this documentation should be in geodesic/module/with-kops

Reply:
Good point